File: Swap-Closet-V4/frontend/src/app/modelos/CartaProductoDTO.ts
export interface CartaProductoDTO {
  //Producto
  productoId: number,
  tipo: string,
  precio: string,
  titulo: string,
  estado: string,
  talla: string,
  fechaDevolucion: string,
  fechaCreacion: string,
  activo: boolean,

    //ImagenProducto
  urlImgProducto: string,

    //Usuario
  usuarioId: number,
  nombreUsuario: string,
  apellidosUsuario: string,
  urlImgUsuario: string,
  ubicacionUsuario: string,

    //Raiting
  raiting: number
}

File: Swap-Closet-V4/backend/src/main/java/org/swapcloset/backend/dto/CartaProductoDTO.java
package org.swapcloset.backend.dto;

import lombok.Data;

import java.util.List;

@Data
public class CartaProductoDTO {

    //Producto
    private Integer productoId;
    private String tipo;
    private String precio;
    private String titulo;
    private String estado;
    private String talla;
    private String fechaDevolucion;
    private String fechaCreacion;
    private Boolean activo;

    //ImagenProducto
    private String urlImgProducto;

    //Usuario
    private Integer usuarioId;
    private String nombreUsuario;
    private String apellidosUsuario;
    private String urlImgUsuario;
    private String ubicacionUsuario;

    //Raiting
    private Double raiting;

}
